docs/cases/cases-overview.asciidoc

```diff
@@ -102,3 +102,45 @@ NOTE: Once a visualization has been added to a case, it cannot be modified or de
   
 [role="screenshot"]
 image::images/cases-open-vis.png[Shows where the Open Visualization option is]
+
+[float]
+[[cases-export-import]]
+== Export and import cases
+
+Cases can be <<cases-export, exported>> and <<cases-import, imported>> as saved objects through the Kibana {kibana-ref}/managing-saved-objects.html[Saved Objects] UI.
+
+IMPORTANT: Before importing Lens visualizations, Timelines, or alerts  into a space, ensure their data is present. Without it, they won't work after being imported.
+
+[float]
+[[cases-export]]
+=== Export a case
+Use the export option to move cases between different Kibana instances. When you export a case, the following data is exported to a newline-delimited JSON (NDJSON) file: case details, user actions, text string comments, case alerts, and lens visualizations.
+
+NOTE: Exported alerts contain metadata that allow the case to reference the appropriate alerts after they're imported. Lens visualizations are exported as JSON blobs.
+
+To export a case:
+
+. Open the main menu, click *Stack Management -> Kibana* and then select the *Saved Objects* tab.
+. Search for the case by choosing a saved object type or by entering the case title in the search bar.
+. Select one or more case and click the *Export* button.
+. Click *Export*. A toast message verifying that your file is downloading displays.
+
++
+TIP: Keep the *Include related objects* option enabled to ensure connectors are exported too.
+
+[role="screenshot"]
+image::images/cases-export-button.png[Shows the export saved objects workflow]
+
+[float]
+[[cases-import]]
+=== Import a case
+
+To import a case:
+
+. Open the main menu, click *Stack Management -> Kibana* and then select the *Saved Objects* tab.
+. Click *Import*.
+. Select the NDJSON file containing the exported case and configure the import options.
+. Click *Import*.
+. Review the import log and click *Done*.
++
+IMPORTANT: If you've imported a case with a connector, you'll be prompted to re-authenticate the connector. To get to your connector, open the main menu, then go to *Stack Management -> Alerts and Insights -> Rules and Connectors -> Connectors*.
```

Comment:
You can also get there by clicking the `Go to connectors` button after the import:

![image](https://user-images.githubusercontent.com/56361221/142671927-591666bd-3a6d-4b81-bdb3-67c95a78e3b6.png)